gm4_tower_structures/data/gm4_tower_structures/functions/spawn/badlands.mcfunction

```diff
@@ -1,3 +1,3 @@
-execute unless block ~ 4 ~ bedrock run summon item ~ ~ ~ {Tags:["gm4_orbis_structure"],Item:{id:"minecraft:stick",Count:1b,tag:{gm4_orbis:{name:"Tall Badlands Tower","structure":{"name":"gm4_tower_structures:badlands",posX:-3,posY:-1,posZ:-3}}}}}
-execute if block ~ 4 ~ bedrock unless block ~ 3 ~ bedrock run summon item ~ ~ ~ {Tags:["gm4_orbis_structure"],Item:{id:"minecraft:stick",Count:1b,tag:{gm4_orbis:{name:"Large Badlands Tower","structure":{"name":"gm4_tower_structures:badlands2",posX:-4,posY:-1,posZ:-4}}}}}
-execute if block ~ 4 ~ bedrock if block ~ 3 ~ bedrock run summon item ~ ~ ~ {Tags:["gm4_orbis_structure"],Item:{id:"minecraft:stick",Count:1b,tag:{gm4_orbis:{name:"Acacia Tower","structure":{"name":"gm4_tower_structures:badlands3",posX:-4,posY:-1,posZ:-4}}}}}
+execute unless block ~ -60 ~ bedrock run summon item ~ ~ ~ {Tags:["gm4_orbis_structure"],Item:{id:"minecraft:stick",Count:1b,tag:{gm4_orbis:{name:"Tall Badlands Tower","structure":{"name":"gm4_tower_structures:badlands",posX:-3,posY:-1,posZ:-3}}}}}
```

Comment:
Can/should this be moved to a loot table? (Apply this comment to the other `/spawn/<biome>` functions)

Reply:
The idea here is that it's seed deterministic by checking the bedrock. But maybe that's not worth it and checking biomes, random chances and heights in a loot table would be a lot cleaner.